### PR TITLE
Adds French translation

### PR DESCRIPTION
**Pull request by cosmo0**

### CODE REVIEW

**Comment by mmatyas** [comment]
Thank you! Just a minor thing, could you merge the two commits into one?

File: pegasus_fr.ts [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+    <name>RebootDialog</name>
+    <message>
+        <location filename="../src/frontend/dialogs/RebootDialog.qml" line="25"/>
+        <source>Reboot</source>
+        <translation>Redémarrer</translation>
+    </message>
+    <message>
+        <location filename="../src/frontend/dialogs/RebootDialog.qml" line="26"/>
+        <source>The system will reboot. Are you sure?</source>
+        <translation>Le système va redémarrer. Êtes-vous sûr ?</translation>
```
Comment (mmatyas) [inline pegasus_fr.ts:219]: There's a space before the `?`, is this correct/intentional?

File: pegasus_fr.ts [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+    <name>ShutdownDialog</name>
+    <message>
+        <location filename="../src/frontend/dialogs/ShutdownDialog.qml" line="25"/>
+        <source>Shutdown</source>
+        <translation>Eteindre</translation>
+    </message>
+    <message>
+        <location filename="../src/frontend/dialogs/ShutdownDialog.qml" line="26"/>
+        <source>The system will shut down. Are you sure?</source>
+        <translation>Le système va s&apos;éteindre. Êtes-vous sûr ?</translation>
```
Comment (mmatyas) [inline pegasus_fr.ts:260]: A space here too.

File: pegasus_fr.ts (hunk 1 above)
Comment (cosmo0) [reply pegasus_fr.ts:219]: Yes there are space before many punctuation points in French, like !, ? or ;.

**Comment by cosmo0** [comment]
Oh I forgot : there are some translations that are marked as unfinished because I didn't find how where they were, mostly in the gamepad configuration. Can you point me to where I can see them in context?

**Comment by cosmo0** [comment]
I should have squashed the commits into one.

**Comment by mmatyas** [comment]
Sure, [here](https://imgur.com/a/no3Jm) are some screenshots, the gamepad screen too. Here's how it looks right now:

pegasus_gamepad_francia

**Comment by cosmo0** [comment]
Thanks! It's not bad actually. I think it's fine :)

**Comment by mmatyas** [comment]
The `unfinished` tags could be removed then, I guess :)

**Comment by cosmo0** [comment]
done

**Comment by mmatyas** [comment]
Thanks!

**Pull request merged**